tests/test_animate.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```

Comment:
This is probably a copy+paste thing, but in general pipx has a bunch of these at the top of files that should be removed. It can be removed here.

Reply:
Ack, yes.  Force of habit!
Done.